src/templates/common/medallia/index.test.tsx

```diff
@@ -20,8 +15,78 @@ jest.mock('@/lib/utils/medallia', () => ({
   setWindowVaSurvey: jest.fn(),
 }))
 
+// Mock recordEvent
+jest.mock('@/lib/analytics/recordEvent', () => ({
+  recordEvent: jest.fn(),
+}))
+
+interface MedalliaEvent extends CustomEvent {
+  detail: {
+    Form_Type?: string
+    Form_ID?: string
+    Feedback_UUID?: string
+    Content?: string
+  }
+}
+
+const mockEvent: MedalliaEvent = {
+  detail: {},
+  initCustomEvent: function (
+    type: string,
+    bubbles?: boolean,
+    cancelable?: boolean,
+    detail?: unknown
+  ): void {
+    throw new Error('Function not implemented.')
+  },
+  bubbles: false,
+  cancelBubble: false,
+  cancelable: false,
+  composed: false,
+  currentTarget: undefined,
+  defaultPrevented: false,
+  eventPhase: 0,
+  isTrusted: false,
+  returnValue: false,
+  srcElement: undefined,
+  target: undefined,
+  timeStamp: 0,
+  type: '',
+  composedPath: function (): EventTarget[] {
+    throw new Error('Function not implemented.')
+  },
+  initEvent: function (
+    type: string,
+    bubbles?: boolean,
+    cancelable?: boolean
+  ): void {
+    throw new Error('Function not implemented.')
+  },
+  preventDefault: function (): void {
+    throw new Error('Function not implemented.')
+  },
+  stopImmediatePropagation: function (): void {
+    throw new Error('Function not implemented.')
+  },
+  stopPropagation: function (): void {
+    throw new Error('Function not implemented.')
+  },
+  NONE: 0,
+  CAPTURING_PHASE: 1,
+  AT_TARGET: 2,
+  BUBBLING_PHASE: 3,
```

Comment:
Are these meaningful? Do we have to test against different values for any reason, or can we just rely on Medallia to do Medallia things and handle all this appropriately?

Reply:
This was another thing that was added with `add missing properties` we don't interact with that part of the event at all. I'm guessing that these are default values. I wouldn't have added them at all if typescript linter didn't cry about it.